frontend-react/src/shared/CodeSnippet/CodeSnippet.module.scss

```diff
@@ -0,0 +1,32 @@
+@use "src/global-modules" as *;
+
+.CodeSnippet {
+    :global {
+        .code_snippet {
+            word-wrap: break-word;
+
+            p {
+                @include u-margin(0px);
+            }
+        }
+
+        .text-highlight {
+            @include u-text("primary-vivid");
+            background-color: transparent;
+        }
+
+        .usa-tooltip__body {
+            @include u-font("sans", "md");
+        }
+
+        .usa-tooltip__body--left {
```

Comment:
is there a way we can avoid using `!important`? feel like a little bit of a code smell to need to use that. from my small amount of tooling around locally, the `grid-row` class on the `<pre>` element in the `<CodeSnippet />` component is causing the inline style to not work for some reason.